src/Form/Modifier/ProductFormModifier.php

```diff
@@ -75,7 +75,12 @@ public function modify(
      */
     private function getBuckarooIdinForProduct(int $productId): ?bool
     {
-        $sql = 'SELECT buckaroo_idin FROM ' . _DB_PREFIX_ . 'bk_product_idin WHERE product_id = ' . (int) $productId;
+        $sql = new \DbQuery();
+
+        $sql->select('buckaroo_idin');
+        $sql->from('bk_product_idin');
+        $sql->where('product_id = ' . pSQL($productId));
```

Comment:
why aren't you using prepared statements?

Reply:
Method 'setParameter' not found in \DbQuery

Reply:
I dont see setParameter.

https://devdocs.prestashop-project.org/8/development/components/database/dbquery/

Reply:
sorry, saw this code https://github.com/buckaroo-it/PrestaShop/blob/develop/src/Repository/PaymentMethodRepository.php#L43-L44, but from what I see this uses the entity manager query builder not the \DbQuery class